samples/previews/azure-functions/service-bus/ASBFunctions_0/AzureFunctions.ASBTrigger.FunctionsHostBuilder/AzureFunctions.ASBTrigger.FunctionsHostBuilder.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\AzureFunctions.ASBTrigger.Static\CustomComponent.cs" Link="CustomComponent.cs" />
```

Comment:
we can also include the message handlers to remove duplicates. See https://github.com/Particular/docs.particular.net/pull/5115